### PR TITLE
Trivial fix for the output of "snappy list"

### PR DESCRIPTION
**Pull request by mvo5**
Instead of:
 Reboot to use 2015.11.24-1 version ubuntu-core.
it will print
 Reboot to use ubuntu-core version 2015.11.24-1.

### CODE REVIEW

File: cmd/snappy/cmd_list.go [hunk 1]
```diff
@@ -133,7 +133,7 @@ func showRebootMessage(installed []snappy.Part, o io.Writer) {
 		}
 
 		// TRANSLATORS: the first %s is a pkgname the second a version
-		fmt.Fprintln(o, fmt.Sprintf(i18n.G("Reboot to use %s version %s."), part.Version(), part.Name()))
+		fmt.Fprintln(o, fmt.Sprintf(i18n.G("Reboot to use %s version %s."), part.Name(), part.Version()))
```
Comment (zyga) [inline cmd/snappy/cmd_list.go]: Unless go's printf supports reordering this will be a pain for translators. This is totally unrelated the fix obviously. Just pointing it out with my i18n hat on.

**Comment by zyga** [comment]
+1

File: cmd/snappy/cmd_list.go (hunk 1 above)
Comment (mvo5) [inline cmd/snappy/cmd_list.go]: Yeah, this is a valid concern. Fortunately go does support reordering, see https://play.golang.org/p/O8krl0c0Us
